reports/Washington.md

```diff
@@ -357,3 +357,51 @@ id: wa-seattle-23
 
 * https://twitter.com/sre_li/status/1278856204018565120
 * https://twitter.com/1stRoundFinish/status/1278854169743388672
+
+
+### Violent arrests made at protests near CHOP | July 2nd
+
+Footage beginning at 14:00 shows police begin to advance on protestors without warning. They use their shields and batons to strike and shove protestors. When protestors become agitated, police tackle protestors and discharge pepper spray. Footage shows police kneeling on one protestor, applying pressure to his neck (allegedly with their forearm) to force him to release his bike.
+
+The second link shows a still of the bicyclist's arrest.
+
+tags: arrest, pepper-spray, spray, strike, beat, shove, tackle, baton, shield, neck, knee, protestor
+
+id: wa-seattle-24
+
+**Links**
+
+* https://www.facebook.com/joeywieser/videos/10158221953345491/
+* https://twitter.com/spekulation/status/1278593887410372608
+
+
+### Police make violent arrest | July 2nd
```

Comment:
I'm like 85% sure this is a separate incident from the previous. But I could be wrong

Reply:
Okay I'm now 100% sure